inc/parser/xhtml.php

```diff
@@ -1108,7 +1108,13 @@ function emaillink($address, $name = null, $returnonly = false) {
     function internalmedia($src, $title = null, $align = null, $width = null,
                            $height = null, $cache = null, $linking = null, $return = false) {
         global $ID;
-        list($src, $hash) = explode('#', $src, 2);
+
+        if (strpos($src, '#') !== false) {
+            list($src, $hash) = explode('#', $src, 2);
+        } else {
+            $hash = null;
+        }
+
```

Comment:
When using a plugin like indexmenu there is no # in $src, which results in an "Undefined offset: 1" error on $hash when using list(), so it is not explode causing the error, it is list().